### ISSUE TITLE
Smartphones shouldn't have UPS

### ISSUE THREAD

**Issue by CoroNaut** [CleverRaven/Cataclysm-DDA#76339]
### Is your feature request related to a problem? Please describe.

Consider the description of the UPS:

https://github.com/CleverRaven/Cataclysm-DDA/blob/98be1d4185bb06b07b18929057fef378eefd2f9e/data/json/items/tool/electronics.json#L688

"It is a device developed jointly by military and scientific interests for use in combat and the field."

Why would the smartphone, held by the vast majority of the population, use the UPS to charge their phone? The life of these citizens was so fine that they all just had smartphones to worry about being charged, and basically nothing else. The zombie apocalypse made the UPS useful for the regular citizen by giving power to a wide range of tools if equipped with a ups conversion mod. This makes it much more valuable.

Tools in a construction site aren't fed from a UPS IRL either, instead they are plugged into nearby buildings or a generator, so there is no argument there.

Firefighters occasionally spawn with the UPS, but don't actually have any tools to attach it to.

### Solution you would like.

We should remove the `USE_UPS` tag from smartphones. The smartphone should just take a standard battery like the ones we have IRL. Instead of UPS, perhaps it could have a tag `INDUCTION_CHARGE` and we could add a special induction charger holder or puck to that the smartphone could be 'attached' too. We have this IRL and it makes much more sense. 

Perhaps for the military population, they could have a lore-friendly communication system, perhaps satellite phones issued to all military personnel. These could have UPS mod. It would make them easy to charge when back at base, or in wartimes when a group of soldiers need some quick and easy power. Scientists would only use UPS in their facility, where, once again, there is mainly tools and equipment.

Firefighter zombies shouldn't have UPS. Unless we give them all UPS tools instead, which is also a cool idea, and would make sense.

https://github.com/CleverRaven/Cataclysm-DDA/blob/ab8659509b83c885b20c87c70f78a55871fbe4ac/data/json/items/tool/electronics.json#L623

### Describe alternatives you have considered.

1. Just going with the smartphone being able to be charged by UPS

### Additional context

This discussion was inspired by #76331

**Comment by Psithief** [comment]
> The smartphone should just take a standard battery like the ones we have IRL.

The smartphones we have in real life these days generally do not have removable batteries.

My understanding is that UPS is just a battery-powered box that has wall-sockets of various kinds, and has some in-built transformers and can generate AC power from DC. It may or may not have USB connectors for charging as well.

Marking the smartphone as 'using UPS' means it can be plugged in to a UPS without input from the player, but it's not clear if this is by USB directly or by a "wall charger". 

Convenient **if** you want to extend the battery-life of your phone, e.g. if using it as a light source for crafting. But on a practical level, sometimes I don't want all the phones I pick up from the ground to drain charge from my UPS without asking.

**Comment by CoroNaut** [comment]
Ah, if the smartphone battery isn't even removable, then I would suggest it come with a cord like other devices, like `"type":"link_up"` (soldering iron, smart lamp, etc) or induction charge flag or both even.

If the UPS is indeed a box with all kinds of wall sockets for tons of devices, why is the smart phone the only device able to link up to it? There is the ups conversion device, which enables most tools to charge from it. The description of the ups conversion mod being:
https://github.com/CleverRaven/Cataclysm-DDA/blob/dd272dd1852db13f7f4b11af861a06dccf6c8d34/data/json/items/toolmod.json#L20

"replaces conventional battery connections with a connector port that **can** plug into a UPS". it doesn't make sense that the smartphone just so happens to have a cable compatible with it.

My thoughts on the UPS was that it was a device that had a bunch of a single socket, and the ups conversion device was just a short cord to link up with it. I bring up my military example again, its a **universal** power supply, it's not very worth it to carry one with you at all times just to extend the battery life of your smart phone. But the military? A single UPS in a group of 10 soldiers makes its benefit extremely worth while. All their devices can link up without worry about them finding many different types of batteries in the wild or when they return to base for new batteries.

**Comment by KheaHyena** [comment]
I do agree here, I don't really see the reason for a smartphone to have an UPS system, specially with all the changes, although I don't see a severe issue with them.
Smartphones do have a cord which you can use to connect to any grid, including one's vehicle, so there's no problem there. Only pet-peeve I have at the moment with them is that it disassembles for a meager light battery (a non-rechargeable one, in the current experimental).

I always saw the UPS as one of those fancy-schmancy Magnetic Rechargers upped to 11 with all the special technology development (still waiting for the Satellite version), so you don't have to worry with attachments and cords.
One can argue they just took advantage of this for a new Smartphone sales-pitch, but if that would be so we should definitely see a lot more tools with it (which would definitely be nice to see, even if only for rescue services or military).

**Comment by kevingranade** [comment]
I agree that the UPS link is tenuous at best, recharging should either be with an item that lets you tether it to a power grid or via a charger item (that is assumed to have the needed cables).  Integrated batteries are the way to go for modern smartphones, even if they were removable it's been a very long time since smartphones ran on consumer batteries instead of bespoke battery packs.

**Comment by Satellite-SLickers** [comment]
as someone who owns a power brick that i just keep in my bag to charge my everything i need. i always assumed that the ups was one of those, but beefier. a big old battery unit with power adapters for any device you might have. if the ups isnt meant to just be a beefier version of the regular commercially available power bricks you can buy at any store, im not actually sure what it even is

**Comment by kevingranade** [comment]
It's not a coherent anything, it's a code-centric solution to some problems we had about 10 years ago that has long since outlived it's usefulness and just causes problems now.  Now we have devices that are (at least moving toward) having reasonably representative battery lifes, and the UI for charging and replacing batteries etc isn't absolutely terrible.
